client/src/Movies/SavedList.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import { NavLink, Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
I know this was given to you initially but as we add our own code we should always try & modify the rest of the code to fit our changes

Reply:
or we could have made use of this class component & implemented a pure component here ?